src/lib.rs

```diff
@@ -0,0 +1,10 @@
+mod big_int;
+mod macros;
+
+#[derive(Debug)]
+pub enum Errors {
+    ConversionError,
+}
+
+pub type BigInt = big_int::BigInt;
+pub type RNSRepresentation = big_int::RNSRepresentation;
```

Comment:
instead of adding type aliases, it is possible to directly export types by using `pub use`:
```suggestion
pub use big_int::BigInt;
pub use big_int::RNSRepresentation;
```